microsoft-365/commerce/subscriptions/move-users-different-subscription.md

```diff
@@ -44,11 +44,9 @@ You must be a Global, License, or User admin to assign licenses. For more inform
 
 1. In the admin center, go to the **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=834822" target="_blank">Active users</a> page.
 2. Select the circles next to the names of the users that you want to replace existing licenses for.
-3. At the top, select **More options (...)**, then select **Manage product licenses**.
-4. In the **Manage product licenses** pane, select **Replace existing product license assignments** \> **Next**.
-5. Switch the toggle to the **On** position for the licenses that you want to assign to these users.\
-    You can limit which services are available to the users. Switch the toggles to the **Off** position for the services that you don't want those users to have. Any previous license assignments for the selected users are removed.
-6. At the bottom of the **Replace existing products** pane, select **Replace** \> **Close**.
+3. At the top, select **Manage product licenses**.
+4. In the **Manage product licenses** pane, Select the licenses that you would like to assign to the user and Unselect the current sibscription.
```

Comment:
```suggestion
4. In the **Manage product licenses** pane, select the licenses that you would like to assign to the user and unselect the current subscription.
```

Reply:
Shouldn't the current subscription be unselected before the new licenses are assigned?